cmd/bblfsh/server.go

```diff
@@ -2,15 +2,16 @@ package main
 
 import (
 	"net"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
I by no means am an expert, but I believe this to be an idiom called `blank identifier` https://golang.org/doc/effective_go.html#blank

In `import` context it prevents a compiler from complaining on un-used imports. I believe here such import have a side-effect of enabling profiling instrumentation.